File: OOP_C++_JAVA/lab-2.cpp
#include <bits/stdc++.h>
using namespace std;
class SumCalculator
{
public:
    int sum;
    int Sum(int a, int b, int c)
    {
        sum = a + b + c;
        cout << "The sum of " << sum << endl;
    }
    
};
int main()
{
    int n1, n2, n3;
    cout << "Enter the three number" << endl;
    cin >> n1 >> n2 >> n3;
    SumCalculator obj;
    obj.Sum(n1, n2, n3);
    

    return 0;
}

File: OOP_C++_JAVA/lab2.java
import java.util.Scanner;

class Sum {
    public int sum, a, b, c;

    Sum(int a, int b, int c) {
        sum = a + b + c;
    }

    void display() {
        System.out.println("The sum is " + sum);
    }
}

public class lab2 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter three number");
        int n1 = scanner.nextInt();
        int n2 = scanner.nextInt();
        int n3 = scanner.nextInt();
        Sum obj = new Sum(n1, n2, n3);
        obj.display();
        scanner.close();

    }
}
